python/verticox/aggregator.py

```diff
@@ -77,6 +78,8 @@ def fit(self):
             logger.info('\n\n----------------------------------------\n'
                         '          Starting new iteration...'
                         '\n----------------------------------------')
+            info(f'Starting iteration num {self.num_iterations}')
```

Comment:
`logger.info()` is for regular python logging. `info()` is the one that shows up in the vantage6 logs.
I'm not yet ready to let go of the regular logging because my awesome dashboard depends on it.